tests/ui_tests/fixtures.json

```diff
@@ -2058,21 +2054,21 @@
 "TTui2_bitcoin-test_signtx_taproot.py::test_send_mixed": "b95105e2678b621a06ffe3c126921a5d89e655b11957e8c76c192b3ce645cf39",
 "TTui2_bitcoin-test_signtx_taproot.py::test_send_p2tr": "fcc17bdfe319aa8f27f01bdb9660b0a8d1dca5a0be74e353bf13a3c2eef39328",
 "TTui2_bitcoin-test_signtx_taproot.py::test_send_two_with_change": "8c1992cb4bed58e0faad9f1a24269866e2d5e5c12106a57c3b8637edd6554e92",
-"TTui2_bitcoin-test_verifymessage.py::test_message_grs": "d9789e293d71b497430538be4f3635c6ec96cd332c30c05676abd390300b2e98",
-"TTui2_bitcoin-test_verifymessage.py::test_message_long": "041181a31c9232d8706eca4541fe5faa264acc43c2d8a6873f382bd67b04f1a6",
-"TTui2_bitcoin-test_verifymessage.py::test_message_testnet": "cf5fc04107faef7b618732d49ed9a9507889b258bc09990b78e32f3e4af396d5",
-"TTui2_bitcoin-test_verifymessage.py::test_message_verify": "b923b1a2b88838808460bf38b15bf805841e280a90723a40a7f47b6306628cd2",
-"TTui2_bitcoin-test_verifymessage.py::test_message_verify_bcash": "2ded468d30dde1494fb3f4ad7dab465c17335f94b59bef11713091389fd77e50",
-"TTui2_bitcoin-test_verifymessage.py::test_verify_bitcoind": "e924ff7a3008c7dca542e36f0026dd418352fb6910ab48dfadc237c36b46ddd3",
-"TTui2_bitcoin-test_verifymessage.py::test_verify_utf": "8cc659614a453c5643e9c211fb070d58e93b472ebe45096c219058170caa3cd2",
-"TTui2_bitcoin-test_verifymessage_segwit.py::test_message_long": "46cbf98f28bb01e4d92acc2755c24bed3616cbf9a189bb9ca0d60cd0af710d8a",
-"TTui2_bitcoin-test_verifymessage_segwit.py::test_message_testnet": "ee70f4a3410c5cb37a351587b38a625b489a801be0f5040fc3649c9b0fb7544d",
-"TTui2_bitcoin-test_verifymessage_segwit.py::test_message_verify": "b934503757f3974319ce43f4f9fad5e9d48291c17892b939bd4548ca122f4f76",
-"TTui2_bitcoin-test_verifymessage_segwit.py::test_verify_utf": "5daff0a6e6df88737ca1b479afd87df2e06e6e2233b86d9da123e2af8a1cecc3",
-"TTui2_bitcoin-test_verifymessage_segwit_native.py::test_message_long": "00c2b350d781b89b7a6df32b337798d85f54b76dd32c1faadd868cf90bd8249e",
-"TTui2_bitcoin-test_verifymessage_segwit_native.py::test_message_testnet": "dd9c50a6fb92ca728fe254111ef0a2d0392cc81ac21dd1ebc009314f24f07d0e",
-"TTui2_bitcoin-test_verifymessage_segwit_native.py::test_message_verify": "dfc35dc605f15bb3f391ae5c10870b3358ea49d1aea02ff9f3607798c15ca667",
-"TTui2_bitcoin-test_verifymessage_segwit_native.py::test_verify_utf": "91b659369711ecee12e7976ad56f6f1b8b748871be5f0d864b38c626109b5ce0",
+"TTui2_bitcoin-test_verifymessage.py::test_message_grs": "2064ccc2b13792fad9d0b5fe7f440dde0c4008e9bdb97836d6daa25abdd3b1f8",
```

Comment:
something very strange is happening here. probably some sort of rebase artifact?

Reply:
No clue where this came from. Reverted in https://github.com/trezor/trezor-firmware/pull/2507/commits/b4e4326e6dd8bf0993bed576f4c7d6acd1fb7240.